src/billing/BillingAccount/BillingAccountControls.test.tsx

```diff
@@ -3,8 +3,8 @@ import userEvent from '@testing-library/user-event';
 import React, { ReactNode, useState } from 'react';
 import { hasBillingScope } from 'src/auth/auth';
 import { BillingAccountControls } from 'src/billing/BillingAccount/BillingAccountControls';
+import { GCPBillingProject, GoogleBillingAccount } from 'src/billing-core/models';
```

Comment:
Within `src/billing` in particular, I've standardized on referencing these models from `src/billing-core/models` instead of the ajax directory.